war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -128,7 +131,7 @@ var createPluginSetupWizard = function() {
 
 	// Instantiate the wizard panel
 	var $wizard = $(pluginSetupWizard());
-	$wizard.appendTo('body');
+	$wizard.appendTo(appendTo);
```

Comment:
Okay, how about calling it something more explicit e.g. `appendToTarget` or `appendToElement` ?